rust/kcl-lib/src/parsing/ast/types/mod.rs

```diff
@@ -1959,6 +1959,10 @@ impl CallExpressionKw {
     fn rename_identifiers(&mut self, old_name: &str, new_name: &str) {
```

Comment:
Should `replace_value()` above also recurse into the unlabeled?

Reply:
oh yeah good catch